docs/get-started/get-started.md

```diff
@@ -239,4 +242,172 @@ CompositionLocalProvider(LocalGravatarTheme provides object : GravatarTheme {
+            email = Email("{USER_EMAIL}")
+            avatarPickerContentLayout = AvatarPickerContentLayout.Horizontal
+        },
+        authenticationMethod = AuthenticationMethod.OAuth(
+            OAuthParams {
+                clientId = "{YOUR_CLIENT_ID}"
+                redirectUri = "{YOUR_REDIRECT_URL}" // In our example this would be https://gravatar.com/redirect_url
+            },
+        ),
+        onAvatarSelected = { avatarUpdateResult -> ... },
+        onDismiss = { gravatarQuickEditorDismissReason ->
+            showBottomSheet = false
+            ...
+        },
+    )
+}
+```
+
+With the provided `clientId` and the `redirectUrl` Quick Editor can launch and handle the full OAuth flow. Once obtained the token will be stored in an encrypted Data Store.
+This token will be later used in subsequent Quick Editor launches to make the user experience more seamless by not having to go through the OAuth flow each time.
+
+When the user logs out form the app, make sure to run:
+
+```kotlin
+GravatarQuickEditor.logout(Email("{USER_EMAIL}"))
+```
+
+Data Store files are subject to Android backups. Because we use encryption, this won't work when restoring the backup on a different device so we have to exclude those files.
+If your app has backup rules configured, those that are provided in the SDK won't be used so you have to copy them to your files.
+
+In `AndroidManifest.xml` add those lines:
+
+```xml
+<application
+        android:allowBackup="true"
+        android:dataExtractionRules="@xml/data_extraction_rules"
+        android:fullBackupContent="@xml/backup_rules"
+        ...>
+```
+
+Content of the [@xml/data_extraction_rules](https://github.com/Automattic/Gravatar-SDK-Android/blob/trunk/gravatar-quickeditor/src/main/res/xml/data_extraction_rules.xml)
+
+```xml
+<?xml version="1.0" encoding="utf-8"?>
+<data-extraction-rules>
+    <cloud-backup>
+        <exclude
+            domain="sharedpref"
+            path="__androidx_security_crypto_encrypted_file_pref__.xml" />
+        <exclude
+            domain="file"
+            path="datastore/quick-editor-preferences.preferences_pb" />
+    </cloud-backup>
+    <device-transfer>
+        <exclude
+            domain="sharedpref"
+            path="__androidx_security_crypto_encrypted_file_pref__.xml" />
+        <exclude
+            domain="file"
+            path="datastore/quick-editor-preferences.preferences_pb" />
+    </device-transfer>
+</data-extraction-rules>
+```
+
+Content of the [@xml/backup_rules](https://github.com/Automattic/Gravatar-SDK-Android/blob/trunk/gravatar-quickeditor/src/main/res/xml/backup_rules.xml)
+
+```xml
+<?xml version="1.0" encoding="utf-8"?>
+<full-backup-content>
+    <exclude
+        domain="sharedpref"
+        path="__androidx_security_crypto_encrypted_file_pref__.xml" />
+    <exclude
+        domain="file"
+        path="datastore/quick-editor-preferences.preferences_pb" />
+</full-backup-content>
+```
```

Comment:
This is technically optional because the SDK will catch an exception and clean the old files from the backup. Should we hide it/move it?

Reply:
If we are ready to handle it internally, I think we can hide this because it makes the SDK more challenging to integrate. If we don't crash, the final result will be the same, right?

Reply:
Yes, although I think excluding those files from the backup is a better practice.

Reply:
I put that in the <details> tag.